pyproject.toml

```diff
@@ -39,9 +39,13 @@ Documentation = "https://ml.energy/zeus"
 [project.optional-dependencies]
 # One day FastAPI will drop support for Pydantic V1. Then fastapi has to be pinned as well.
 perseus = ["fastapi[all]", "pydantic<2", "lowtime", "aiofiles", "httpx"]
+bso = ["pydantic<2", "httpx"]
+bso-server = ["fastapi[all]","sqlalchemy","pydantic<2"]
+migration = ["alembic", "sqlalchemy", "pydantic<2", "python-dotenv", "cryptography"]
```

Comment:
What's cryptography?

Reply:
Ah, I think I can remove that. It was for MySQL. https://stackoverflow.com/questions/54477829/cryptography-is-required-for-sha256-password-or-caching-sha2-password